mechanics.html

```diff
@@ -153,7 +153,7 @@ <h1>Handling</h1>
     <p><em>also see:</em></p>
 <ul>
 	<li><a class="lna" href="https://www.youtube.com/watch?v=rKQZdRu6_g0" title="How to properly set your TETR.IO Handling!">25Pi25's YouTube video</a> summarizing how to set your handling.</li>
-	<li><a class="lna" href="https://cdn.discordapp.com/attachments/813901028394795078/871859207605985300/DAS_and_ARR.gif">This instinctual <code>.gif</code> file</a> everyone keeps sharing to which I cannot attribute to anyone at this point.</li>
+	<li><a class="lna" href="https://zudo.pics/342h1qfh.gif">This instinctual <code>.gif</code> file</a> everyone keeps sharing to which I cannot attribute to anyone at this point.</li>
```

Comment:
not a perm fix since discord links are ephemeral now, swapped it to my own img host for the pr